tidb-cloud/migrate-from-mysql-using-aws-dms.md

```diff
@@ -96,9 +96,7 @@ AWS DMS does not support replicating `DROP TABLE`.
 
     ![Copy the public and private network IP addresses](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-copy-ip.png)
 
-2. Go to the [TiDB Cloud console](https://tidbcloud.com/console/clusters), locate the target cluster, and click **Connect** to get the TiDB Cloud database connection information.
-
-    ![Get the TiDB Cloud database connection information](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-connect.png)
+2. In the TiDB Cloud console, go to the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target cluster, and then click **Connect** in the upper-right corner to get the TiDB Cloud database connection information.
 
 3. Under **Step 1. Create traffic filter**, click **Edit**, enter the public and private network IP addresses that you copied from the AWS DMS console, and then click **Update Filter**. It is recommended to add the public IP address and private IP address of the AWS DMS replication instance to the TiDB cluster traffic filter at the same time. Otherwise, AWS DMS might not be able to connect to the TiDB cluster in some scenarios.
 
```

Comment:
L103 `![Update the TiDB Cloud traffic filter](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-traffic-filter.png)`

To protect sensitive information, the cluster name in the screenshot should be obscured. Another option is to remove the screenshot entirely.

<img width="867" alt="image" src="https://user-images.githubusercontent.com/60599231/210474382-ac351930-5b30-45a2-b492-8341bda010e1.png">

Reply:
+1